src/PointwiseFunctions/Hydro/EquationsOfState/EquationOfState.hpp

```diff
@@ -646,7 +646,42 @@ class EquationOfState<IsRelativistic, 3> : public PUP::able {
    * \f].
    * With \f$p, e\f$ the pressure and energy density respectively,
    * \f$s\f$ the entropy density, \f$Y_e\f$ the electron fraction
-   * and \f$\rho\f$ the rest-mass density.
+   * \f$\rho\f$ the rest-mass density, and \f$h\f$ the enthalpy density.
+   * Note that \f$e\f$ is the total energy density and not the internal energy,
+   * therefore
+   * \f[
+   * \frac{\partial p}{\rho} |_{e, Y_e} \neq \chi \equiv \frac{\partial
```

Comment:
Should the `\rho` in the denominator be `\partial \rho` (also on the next line)? Also in the first equation above